sem_covid/entrypoints/etl_dags/eurlex_covid19.py

```diff
@@ -508,16 +508,16 @@ def download_dataset_items(json_file_name):
         else:
             logger.exception(f"No manifestation has been found for {item['title']}")
 
-    minio.put_object_from_string(json_file_name, json.dumps(json_content))
+    minio.put_object(json_file_name, json.dumps(json_content))
 
     logger.info(f"Downloaded {counter['html']} HTML manifestations and {counter['pdf']} PDF manifestations.")
 
 
 def extract_content_with_tika(json_file_name):
     logger.info(f'Using Apache Tika at {config.APACHE_TIKA_URL}')
     logger.info(f'Loading resource files from {json_file_name}')
-    minio = MinioAdapter(config.EU_CELLAR_BUCKET_NAME, config.MINIO_URL, config.MINIO_ACCESS_KEY,
-                         config.MINIO_SECRET_KEY)
+    minio = MinioObjectStorage(config.EU_CELLAR_BUCKET_NAME, config.MINIO_URL, config.MINIO_ACCESS_KEY,
```

Comment:
shall we build a registry (similar to the data sources) for this? Then accessing the buckets for each pipeline will be even simpler.